index.html

```diff
@@ -3549,28 +3548,45 @@ <h3><dfn>Writing or pushing content</dfn></h3>
         the <a>activated reader objects</a>, run the following sub-steps:
         <ol>
           <li>
-            If |reader|.<a>[[\Id]]</a> is [= dictionary member/present =]
-            and it is not equal to any |record|'s
-            <a href="#dom-ndefrecord-id">id</a> where |record|
-            is an element of |message|, [= iteration/continue =].
-          </li>
-          <li>
-            If |reader|.<a>[[\RecordType]]</a> is [= dictionary member/present =]
-            and it is not equal to any |record|'s
-            <a href="#dom-ndefrecord-recordtype">recordType</a> where |record|
-            is an element of |message|, [= iteration/continue =].
-          </li>
-          <li>
-            If |reader|.<a>[[\MediaType]]</a> is not `""` and
-            it is not equal to any |record|'s
-            <a href="#dom-ndefrecord-mediatype">mediaType</a> where |record| is
-            an element of |message|, [= iteration/continue =].
-          </li>
-          <li>
-            <a>Fire an event</a> named "`reading`" at |reader|
-            using <a>NDEFReadingEvent</a> with its <a>serialNumber</a> attribute
-            initialized to |serialNumber| and <a>message</a> attribute
-            initialized to |message|.
+            [= list/For each =] |record:NDEFRecord| in |message:NDEFMessage|,
+            <ol>
+              <li>
+                Let |matched:boolean| be `false`.
+              </li>
+              <li>
+                If |reader|.<a>[[\Id]]</a> is not `""` and is not equal to
```

Comment:
Shall we use "empty string" instead of double quotes as in initial value?
![image](https://user-images.githubusercontent.com/634478/71342378-e0fb3580-255c-11ea-99e0-f3a2e5e3f490.png)

Reply:
I noticed there's some inconsistency in defining `Id`.

First we need to decide `Id` should initially be `undefined` or `An empty string`, then we need to update the following 3 places to be consistent:

1) https://w3c.github.io/web-nfc/#the-ndefscanoptions-dictionary
> dictionary NDEFScanOptions {
  USVString id;
  USVString recordType;
  USVString mediaType = "";
  AbortSignal? signal;
};

2) https://w3c.github.io/web-nfc/#dom-ndefscanoptions-id
> If the dictionary member is not present, then it will be ignored by the NFC listen algorithm.

3)  **here, this matching algorithm**.

Reply:
Let's track this in a separate issue.

Reply:
See #488